src/scripts/main.js

```diff
@@ -1,5 +1,20 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+const coordsWall = wall.getBoundingClientRect();
+
 document.addEventListener('click', (e) => {
-  // write code here
+  const coordX = e.pageX - coordsWall.left - 25;
+  const coordY = e.pageY - coordsWall.top - 25;
+
+  if (coordX > coordsWall.height || coordX < 0) {
```

Comment:
The condition `coordX > coordsWall.height` is incorrect for checking horizontal boundaries. It should be `coordX > coordsWall.width` to ensure the X coordinate is within the wall's width.